docs/main/guides/autofill-credentials.md

```diff
@@ -83,13 +83,14 @@ You can do this in your  `capacitor.config.ts` or `capacitor.config.json` file:
 const config: CapacitorConfig = {
 ...
   server: {
-    hostname: 'my-app.com'
+    hostname: 'my-app.com',
+    androidScheme: 'https',
   }
 };
 ```
 
-:::warning
-Changing the `hostname` from the default of `localhost` affects the use of Web APIs that would otherwise require a [secure context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) such as [`navigator.geolocation`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/geolocation) and [`MediaDevices.getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia). Be sure to test this type of functionality after making the change.    
+:::note
```

Comment:
I think the whole note should be removed, as for making autofill work https has to be used, so it becomes irrelevant and can confuse iOS users.

Reply:
Cool, i'll just remove it. As long as they follow the guide they shouldn't run into a problem with secure contexts